src/components/strategies/overlapping/OverlappingAction.tsx

```diff
@@ -20,7 +19,7 @@ interface Props {
   anchor: 'buy' | 'sell';
   action?: BudgetAction;
   setAction: (action: BudgetAction) => void;
-  error: string;
+  error?: string;
```

Comment:
The error prop is never set outside of the OverlappingAction component

Reply:
removed unused prop